components/Card.vue

```diff
@@ -1,13 +1,34 @@
 <template>
-  <v-card :to="`../item/${item.Id}`">
-    <v-img class="cardImage" :src="imageLink(item.Id)" />
-    <v-card-title>
-      <span>{{ item.Name }}</span>
-    </v-card-title>
-    <v-card-subtitle>
-      <span>{{ item.ProductionYear }}</span>
-    </v-card-subtitle>
-  </v-card>
+  <nuxt-link
+    :to="`/item/${item.Id}`"
+    style="text-decoration: none; color: inherit"
+  >
+    <div class="card-box">
+      <div :class="cardType">
+        <button
+          class="card-content card-content-button d-flex justify-center align-center"
+          :style="{ backgroundImage: `url('${imageLink(item.Id)}')` }"
+        >
+          <v-icon
+            v-if="!item.ImageTags.Primary"
+            size="96"
+            color="grey lighten-3"
+          >
+            {{ itemIcon }}
+          </v-icon>
+        </button>
+        <div class="card-overlay d-flex justify-center align-center">
+          <v-btn fab color="primary" :to="`/item/${item.Id}/play`">
+            <v-icon size="36">mdi-play</v-icon>
+          </v-btn>
+        </div>
+      </div>
+      <div class="card-text">
+        <div class="card-title mt-1">{{ item.Name }}</div>
+        <div class="card-subtitle grey--text">{{ item.ProductionYear }}</div>
```

Comment:
spans

Reply:
Imo, I'd rather keep the templates cleaner than really care about how the dom looks.

Having spans everywhere, in addition to the existing templating, will not only get confusing but it also makes everything harder to read.

Reply:
Ok, sounds good to me.